db/schema.rb

```diff
@@ -336,6 +336,9 @@
     t.integer  "stage_id",    limit: 4,     null: false
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.boolean  "before_deploy",               default: false, null: false
```

Comment:
Because the migration file was generated two weeks ago, the schema version (aka the timestamp) is now not the latest, due to other PRs merged. If we haven't deployed to production, since two weeks ago, it's probably fine. Otherwise, I could re-generate the migrations file to give it a newer timestamp.